ldm/invoke/model_cache.py

```diff
@@ -145,10 +145,14 @@ def list_models(self) -> dict:
             else:
                 status = 'not loaded'
 
-            models[name]={
-                'status' : status,
-                'description' : description
-            }
+            models.update(
+                {
+                    name: {
```

Comment:
The syntax is a new dict that overwrites the existing keys, so this needs quotes.

```suggestion
                    'name': {
```

Reply:
I personally don't understand the original change, or psychedelicious's suggestion.
In my opinion, the original change is switching between equivalent ways of updating a dict (the pre-commit being more readable than the commit).
psychedelicious's suggestion changes the code such that the keys to models are only 'name', and its constantly overwritten.
Anyway I suggest adding a code comment explaining what is the exact merit of the change, for future reference.

Reply:
@psychedelicious is correct; this code will throw an unknown key exception without the quotes.

Having said that, they're "a horse apiece" - using Dict's `update()` method is more Pythonic, but I find them equally readable, and equally obvious.

Reply:
Actually @tildebyte I think @mebelz is correct, `name` is a variable in this context - the name of the model to be updated.